unicodetools/src/main/java/org/unicode/xml/GeneratePropertyValues.java

```diff
@@ -123,7 +124,7 @@ public static void main(String[] args) throws Exception {
 
         if (options[HELP].doesOccur) {
             System.out.println(
-                    "GeneratePropertyValuesList --ucdversion {version number} --outputfolder {destination}");
+                    "GeneratePropertyValuesList [--ucdversion {version number}] [--outputfolder {destination}]");
```

Comment:
you added brackets to the outputfolder, and you no longer throw an exception if it's absent, but it still REQUIRES_ARG